valet-key/ValetKey.Web/Controllers/SasController.cs

```diff
@@ -56,7 +57,7 @@ private StorageEntitySas GetSharedAccessReferenceForUpload(string blobName)
         {
             var blob = blobServiceClient.GetBlobContainerClient(this.blobContainer).GetBlobClient(blobName);
 
-            var storageSharedKeyCredential = new StorageSharedKeyCredential(blobServiceClient.AccountName, ConfigurationManager.AppSettings["AzureStorageEmulatorAccountKey"]);
+            var storageSharedKeyCredential = new StorageSharedKeyCredential(blobServiceClient.AccountName, this.configuration.GetSection("AppSettings:StorageKey").Value);
```

Comment:
As mentioned in the readme, this should be using managed identity, and not storage keys